solr/solr-ref-guide/modules/deployment-guide/pages/metrics-reporting.adoc

```diff
@@ -156,6 +156,7 @@ If this section is not defined, the following default configuration is used whic
 </metrics>
 ----
 
+[#the-metrics-reporters-element]
```

Comment:
This and the similar edit in configuring-solr-xml.adoc were done due to IntelliJ throwing errors, that the asciidoc plugin could not find the anchor. So I added these explicit anchors to silence it. Antora build was apparently ok with it.